collect/central_collector.go

```diff
@@ -278,8 +278,10 @@ func (c *CentralCollector) shutdown(ctx context.Context) error {
 			}
 		}
 		// we have to make sure the health check says we're alive but not accepting data during shutdown
-		c.Health.Ready(receiverHealth, false)
-		c.Health.Ready(deciderHealth, false)
+		c.Health.Unregister(receiverHealth)
```

Comment:
Now think about it, this probably should be in `Stop()` instead of this goroutine?